src/all/mangafireto/build.gradle

```
@@ -0,0 +1,13 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    extName = 'MangaFire.to'
```

Make the same name from the source `name` attribute.

```suggestion
    extName = 'MangaFire'
```